src/pages/tutorials/smart-rollups/optimize.md

```diff
@@ -0,0 +1,104 @@
+---
+id: optimize
+title: "Part 3: Optimizing the kernel"
+lastUpdated: 25th October 2023
+---
+
+To originate the kernel on Tezos, it must fit within the maximum size for a layer 1 operation.
```

Comment:
Should we say what this size is? e.g. 
```suggestion
To originate the kernel on Tezos, it must fit within the maximum size for a layer 1 operation (< 32KB).
```

Reply:
"the maximum size for a layer 1 operation (32KB)"